corehq/apps/reports/generic.py

```diff
@@ -943,10 +943,11 @@ def report_context(self):
             context.update(provider_function(self))
         return context
 
-    def table_cell(self, value, html=None):
+    def table_cell(self, value, html=None, zerostyle=False):
+        styled_value = '<span class="gray-zero">0</span>' if zerostyle and value == 0 else value
```

Comment:
+1 about HTML in Python. :-/ Also less than happy with using the same HTML twice in this PR, but couldn't think of a clean way around that. I really like your `muted` suggestion. I'll try that, and drop the change to tables.less.